CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,73 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Let's not copy this over to our repo.  I like linking to the Twilio Labs one because if it ever changes, this won't go stale.